### DISCUSSION TITLE
Is there any benefit or detriment to adding your students to the GH classroom?

### DISCUSSION THREAD

**Discussion by Lvrks** [community/community#64495] [GitHub Education]
### Hi everyone,

Hi Everyone! The title is the same as my question so I'm wondering, just out of curiosity, is there any benefit or detriment to adding your students to the GH classroom roster?

**Comment by arafajs** [comment]
Yes. It can give your students access to a variety of resources, such as code repositories, issue trackers, and wikis.

**Reply by geoffreylgv** [reply]
Hi @HelonDCI, absolutely yes, many advantages there. Exposes students to industry-standard tools, GitHub Classroom enhances collaboration, version control, and coding skills for students, project management skills, and these resources as @arafajs mentioned.
